ledger/src/blockstore_db.rs

```diff
@@ -126,11 +126,22 @@ pub mod columns {
     pub struct Rewards;
 }
 
+pub enum AccessType {
+    OnlyPrimary,
+    AllowSecondary,
+}
```

Comment:
I think the naming of the  `AccessType` enum caused my confusion at https://github.com/solana-labs/solana/pull/10209#issuecomment-636646312

Is this correct?
1. `AccessType::OnlyPrimary` means: only open the database as primary (but other secondary users are permitted)
2. `AccessType:: AllowSecondary` means: try to open the database as primary and if that fails due to a current primary user then then open the database as secondary.

If that's right, these new enum names make things more clear for me personally:
```rust
pub enum AccessType {
    PrimaryOnly,
    TryPrimaryThenSecondary,
}
```

Reply:
@mvines Yeah, that understanding is right! I renamed accordingly; thanks for suggestion!